sixtyfps_compiler/generator/rust.rs

```diff
@@ -1951,3 +1959,21 @@ fn compile_path(path: &Path, component: &Rc<Component>) -> TokenStream {
         }
     }
 }
+
+// In Rust debug builds, accessing the member of the FIELD_OFFSETS ends up copying the
+// entire FIELD_OFFSETS into a new stack allocation, which with large property
+// binding initialization functions isn't re-used and with large generated inner
+// components ends up large amounts of stack space.
+fn access_component_field_offset(component_id: &Ident, field: &Ident) -> TokenStream {
+    quote!({
+        let field = &#component_id::FIELD_OFFSETS.#field;
+        *field
+    })
+}
+
+fn access_self_field_offset(field: &Ident) -> TokenStream {
```

Comment:
is that not the same as `access_component_field_offset(quote!(Self), field)` ?
either just make access_component_field_offset take a TokenStream instead.  or replace quote by format_indent or something

Reply:
Good idea! Done.